### DISCUSSION TITLE
GUI-Key Position on tiling window manager

### DISCUSSION THREAD

**Discussion by mbauhardt** [manna-harbour/miryoku#65] [Q&A] [answered]
Hi,
I have an keyboardio atreus, and I like this layout. But somehow I feel the bottom row can be shrinked to have only 3 keys like on the corne keyboard. After some googling hi found https://precondition.github.io/home-row-mods and stumbled over the miryoku layout. I felt immediately in love with this layout. But before investing time and money into new keyboard and layout I have one question regarding the home row mods especially the GUI key.

I'm on linux incl the window  manager `dwm`. I'm working with home row mods on my atreus since a couple of days. It feels pretty good from the beginning. But I think the Gui key is a bit misleading on the pinky. I believe OSX users having the same issue. But, because I'm on linux, I'm using the CTRL key very often as well as the GUI key.

So has someone of you experience with tiling window managers like dwm, i3 or xmonad?
maybe the GUI key could be on the inner column next to the pointer finger. what do you think? Are there an alternative recommended home row mods combination which make sense from a ergonomic point of view?

**Comment by manna-harbour** [comment, marked as answer]
If you'd like to try it, you can build [Miryoku QMK for Keyboardio Atreus](https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#keyboardioatreus).  Or you could use [Miryoku KMonad](https://github.com/manna-harbour/miryoku_kmonad), you'd just have to create a simple tap-only layout on the Atreus, and add a matching block to the `defsrc` section [here](https://github.com/manna-harbour/miryoku_kmonad/blob/main/src/miryoku_kmonad.kbd.cpp#L40).

The mods were designed with tiling window managers in mind.  It's easy to hold Gui and combine with other mods and any other key.  I haven't had problems with my pinkie for that, and the pinkie column is deprioritised on the rest of the layout.

The least used mod should be on the pinkie though. If you do heavily use Gui and seldom use Alt you might like to swap those.  It'd be a one line change [here](https://github.com/manna-harbour/miryoku_babel/blob/7eb2ea0b5e361e43c2ced1b2273f89fce284b98f/readme.org?plain=1#L110), and [easy customisation of mods is a planned feature](https://github.com/manna-harbour/miryoku/issues/33).

If you have Gui somewhere other than on its own finger you won't be able to combine with other mods.  E.g. with Gui on inner index you won't be able to combine with Shift.

If you don't use all the extra layers, another approach is to replace the finger hold with a thumb movement by adding a layer where everything has Gui already applied, in the same way as Sym is just Num + Shift.

**Reply by mbauhardt** [reply]
thx for your reply. didn't hear hear from kmonad before. will take a look into.
my question was more theoretical not practical e.g. which tool could help.
so i was more interested into personal experience or thoughts about using home-row-mods on linux and tiling WM where CTRL and GUI is used a lot. what is the most comfortable position for these keys from an ergonomic point of view.
maybe I have to play around and feel how it is to switch ALT and GUI for example.
